pkg/engine/engine.go

```diff
@@ -158,6 +161,23 @@ func (e *Engine) alterFuncMap(t *template.Template) template.FuncMap {
 		return val, nil
 	}
 
+	// Add the 'tpl' function here
+	funcMap["tpl"] = func(tpl string, vals chartutil.Values) (string, error) {
+		r := renderable{
+			tpl:  tpl,
+			vals: vals,
+		}
+
+		templates := map[string]renderable{}
+		templates["template"] = r
+
+		result, err := e.render(templates)
```

Comment:
When a chart is rendered I add the Chart's templates as the current templates in the engine. If something else is rendered it will add the current templates to the render context.

@technosophos What do you think about it?